pkg/dynamic/client.go

```diff
@@ -106,6 +108,8 @@ func (c *knDynamicClient) ListCRDs(ctx context.Context, options metav1.ListOptio
 		return nil, err
 	}
 
+	uList.SetGroupVersionKind(schema.FromAPIVersionAndKind(listVersion, listKind))
+	uList.GetObjectKind().SetGroupVersionKind(schema.FromAPIVersionAndKind(listVersion, listKind))
```

Comment:
Not sure why this is needed. When setting the GVKs on other entities in this client like in `ListSourcesUsingGVKs` the object kind is not set.

Should we update those places, too ?

Reply:
ok, should have read the PR description first ;-)

Still, I think there might be more places that need an update.

Reply:
So should we postpone the update of object kind at the other places in the dynamic client to post 0.23 ? wdyt ?